resources/definitions/malyan_m200.def.json

```diff
@@ -0,0 +1,90 @@
+{
+    "id": "malyan_m200",
+    "version": 2,
+    "name": "Malyan M200",
+    "inherits": "fdmprinter",
+    "metadata": {
+        "author": "Brian Corbino",
+        "manufacturer": "Malyan",
+        "category": "Other",
+        "file_formats": "text/x-gcode",
+        "platform": "malyan_m200_platform.stl",
+        "has_variants": true,
+        "has_variant_materials": false,
+        "has_materials": true,
```

Comment:
There is no machine specific material profiles so, this should be `false`.

Reply:
Are you talking about ```has_materials```? Setting that to ```false``` will remove all materials. Were you thinking about ```has_machine_materials```?

Reply:
I'll have a look. There is some wierd logic in fetching quality profiles which doesn't get anything if `has_materials` is set to `true` for this printer...

Reply:
Yeah I was confounded by this as well.  Agree with fieldOfView, the machine does have materials but not machine specific materials.  Would be helpful if any of the metadata fields were documented ;)